src/stories/components/Radio.stories.jsx

```diff
@@ -0,0 +1,18 @@
+import Radio from '@components/templates/Radio'
+
+export default {
+  title: 'Component/Radio',
+  component: Radio,
+  argTypes: {
+    style: { control: { type: 'object' } },
+    className: { type: 'text' },
+    name: { type: 'text' },
+    radioDirection: { control: { type: 'text' } },
+  },
+}
+
+const lists = [{ value: 'option1' }, { value: 'option2' }, { value: 'option3' }]
```

Comment:
```suggestion
const lists = [
  { 
    text: '옵션 1',
    value: 'option1',
  },
  { 
    text: '옵션 2',
    value: 'option2',
  },
  { 
    text: '옵션 3',
    value: 'option3',
  },
]
```
필드의 보여지는 텍스트와 api 요청 시, 보낼 속성이 다를 수 있어 name 속성이 추가되면 좋을 것 같습니다! 😊